cpp/tests/neighbors/mg.cuh

```diff
@@ -22,7 +22,7 @@
 #include <cuvs/neighbors/cagra.hpp>
```

Comment:
this file will go away after merged of [this PR](https://github.com/rapidsai/cuvs/pull/869)